packages/calcite-components/src/components/block-section/block-section.e2e.ts

```diff
@@ -233,4 +233,77 @@ describe("calcite-block-section", () => {
     expect(await element.getProperty("open")).toBe(false);
     expect(toggle.getAttribute("aria-expanded")).toBe("false");
   }
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(
+        html`
+          <calcite-block-section text="Planes" open icon-end="pen" icon-start="pen" text="a block-section">
+            <p>Block section content</p>
+          </calcite-block-section>
+        `,
+        {
+          "--calcite-block-section-border-color": {
+            targetProp: "borderBlockEndColor",
+          },
+          "--calcite-block-section-background-color": [
+            {
+              targetProp: "backgroundColor",
+            },
+            {
+              shadowSelector: `.${CSS.toggle}`,
+              targetProp: "backgroundColor",
+            },
+            {
+              shadowSelector: `.${CSS.toggleContainer}`,
+              targetProp: "backgroundColor",
+            },
+          ],
+          "--calcite-block-section-header-text-color": [
+            {
+              targetProp: "color",
+            },
+            // {
+            //   shadowSelector: `.${CSS.toggle}`,
+            //   targetProp: "color",
+            // },
+          ],
+          "--calcite-block-section-text-color": [
+            { shadowSelector: `.${CSS.chevronIcon}`, targetProp: "color" },
+            { shadowSelector: `.${CSS.iconStart}`, targetProp: "color" },
+            // TODO: bug CSS.iconEnd doesn't properly apply on the element
```

Comment:
If this can't be tackled in this PR, could you log a bug for this and remove the commented section? Applies to below.

Reply:
Done: [block-section fails to attach CSS icon--end class](https://github.com/Esri/calcite-design-system/issues/11053#top) #11053